lib/determine-basal/determine-basal.js

```diff
@@ -73,10 +73,13 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
     }
     if (minAgo > 12 || minAgo < -5) { // Dexcom data is too old, or way in the future
         rT.reason = "If current system time "+systemTime+" is correct, then BG data is too old. The last BG data was read "+minAgo+"m ago at "+bgTime;
-    }
     // if BG is too old/noisy, or is completely unchanging, cancel any high temps and shorten any long zero temps
-    if ( glucose_status.short_avgdelta == 0 && glucose_status.long_avgdelta == 0 ) {
-        rT.reason = "Error: CGM data is unchanged for the past ~45m";
+    } else if ( glucose_status.short_avgdelta == 0 && glucose_status.long_avgdelta == 0 ) {
+        if ( glucose_status.last_cal && glucose_status.last_cal < 3 ) {
```

Comment:
probably not a big concern, but does `glucose_status.last_cal < 3`  deal with cgm gaps?